spec/truffle/pre_initialization_spec.rb

```diff
@@ -76,5 +76,13 @@
         ENV.delete var
       end
     end
+
+    it "uses the runtime value of $TZ" do
+      with_timezone("America/New_York") do
+        Time.local(2001, 1, 1).zone.should == "EST"
```

Comment:
I think this spec is going to fail in about 6 months when the timezone switches to "EDT".

Reply:
It's a fixed date, the zone name should not change.